serverless/functions/serverless-functions-getting-started.adoc

```diff
@@ -28,6 +28,8 @@ ifdef::openshift-enterprise[]
 [role="_additional-resources"]
 == Additional resources
 * xref:../../registry/securing-exposing-registry.adoc#securing-exposing-registry[Exposing a default registry manually]
+* link:https://plugins.jetbrains.com/plugin/16476-knative--serverless-functions-by-red-hat[Marketplace page for the Intellij Knative plug-in]
```

Comment:
This link works from here, but not in the preview, so I guess just make sure that it works in the published content. It looks like when the preview is created, it adds some characters to replace the double dashes in the link so I'm worried that this will happen when it's transformed during publishing.

Reply:
This may be helpful: https://docs.asciidoctor.org/asciidoc/latest/macros/complex-urls/

Reply:
Good catch, fixed.